### ISSUE TITLE
0.5.3 release?

### ISSUE THREAD

**Issue by yozlet** [PaesslerAG/django-act-as-auth#26]
@Eorhim - it looks like you attempted to push 0.5.3 to Pypi but it didn't take. Is this something that needs extra help? I ask because the lack of Django 2.2 is a pain. (Yes, I can reference the git repo instead of Pypi, but that's not a long-term solution.)

**Comment by Eorhim** [comment]
Yes, I would appreciate some help. I'm not familiar with the pipeline configuration. I copied the config from a working pipeline and have no idea, why it isn't working here.

**Comment by yozlet** [comment]
OK, is it giving you any particular errors that you can share here? Any other evidence of what's happening, and how you know that it isn't working?
